_rx_tcp_schemes.search: TypeError: expected string or bytes-like object

```python
        if connector is None:
>           if _rx_tcp_schemes.search(docker_host):
E           TypeError: expected string or bytes-like object
```
Probably better to raise an error if `docker_host` is None.

https://github.com/aio-libs/aiodocker/blob/1f3047e3ccbfd8305cfc23962e15b1acbaeeef11/aiodocker/docker.py#L57-L72

--- a/CHANGES/155.bugfix
+++ b/CHANGES/155.bugfix
@@ -1,0 +1,1 @@
+Improve logging when docker socket not available.

--- a/CHANGES/156.bugfix
+++ b/CHANGES/156.bugfix
@@ -1,0 +1,1 @@
+Fix current project version.

--- a/aiodocker/__init__.py
+++ b/aiodocker/__init__.py
@@ -1,7 +1,7 @@
 from .docker import Docker
 
 
-__version__ = '0.8.6'
+__version__ = '0.8.6a0'
 
 
 __all__ = ("Docker", )

--- a/aiodocker/__init__.py
+++ b/aiodocker/__init__.py
@@ -1,7 +1,7 @@
 from .docker import Docker
 
 
-__version__ = '0.8.5a0'
+__version__ = '0.8.6a0'
 
 
 __all__ = ("Docker", )

--- a/aiodocker/docker.py
+++ b/aiodocker/docker.py
@@ -68,6 +68,12 @@ class Docker:
             'Invalid API version format'
         self.api_version = api_version
 
+        if docker_host is None:
+            raise ValueError(
+                "Missing valid docker_host."
+                "Either DOCKER_HOST or local sockets are not available."
+                )
+
         if connector is None:
             if _rx_tcp_schemes.search(docker_host):
                 if os.environ.get('DOCKER_TLS_VERIFY', '0') == '1':

